cosmoz-omnitable-styles.js

```diff
@@ -85,11 +85,14 @@ export default css`
 	}
 	:host a {
 		color: var(--primary-link-color, inherit);
-		text-decoration: var(--cosmoz-omnitable-link-decoration, underline);
+		text-decoration: var(--cosmoz-omnitable-link-decoration, none);
```

Comment:
```suggestion
		text-decoration: var(--cosmoz-omnitable-link-decoration, underline);
```

Keep the default look and adjust in FE.